prototypes/process-explorer/dotnet/src/MorganStanley.ComposeUI.ProcessExplorer.Server/Server/Infrastructure/Grpc/ProcessExplorerMessageHandlerService.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace MorganStanley.ComposeUI.ProcessExplorer.Server.Server.Infrastructure.Grpc;
 
-internal class ProcessExplorerMessageHandlerService : ProcessExplorerMessageHandler.ProcessExplorerMessageHandlerBase
+public class ProcessExplorerMessageHandlerService : ProcessExplorerMessageHandler.ProcessExplorerMessageHandlerBase
```

Comment:
Why did you make this change?

Reply:
@lilla28 do you remember exactly why we made this change?

Reply:
@morganstanley/compose-developers if there're no objections this can be followed up when we implement the side-car for the backend.